L/libCppInterOp/bundled/patches/windows.patch

```diff
@@ -47,3 +47,286 @@ index e25ee9a..6064952 100644
 -- 
 2.39.5 (Apple Git-154)
 
+
+From ce769c6e91730c753e30dfd072a3d991d87e4688 Mon Sep 17 00:00:00 2001
+From: Gnimuc <qiyupei@gmail.com>
+Date: Sun, 12 Jan 2025 19:01:50 +0900
+Subject: [PATCH 2/2] Fix symbol visibility issues for C API
```

Comment:
I presume this second patch should be upstreamed too?

Reply:
Yes. https://github.com/compiler-research/CppInterOp/pull/440

Reply:
Ideally all patches except the cmake patch should be upstreamed.